beacon-node-oapi.yaml

```diff
@@ -421,3 +424,13 @@ components:
       required: true
       schema:
         $ref: '#/components/schemas/ConsensusVersion'
+    Eth-Blinded-Payload:
+      description: Required in response so client can deserialize returned json or ssz data to the correct object.
```

Comment:
```suggestion
      description: Required in response so client can deserialize returned JSON or SSZ data to the correct object.
```

Reply:
The existing `Eth-Consensus-Version` uses lower-case so I stuck with it for consistency.